src/App.tsx

```diff
@@ -14,11 +15,144 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type SortBy = 'none' | 'name' | 'length';
```

Comment:
advise - use enum instead of type with strings
```suggestion
enum SortBy {
  none,
  name,
  length,
}
```

then in evaluations just use `SortBy.name` / `SortBy.none` / `SortBy.length`